source/core/parameters.cc

```diff
@@ -2935,6 +2951,26 @@ namespace Parameters
           Patterns::Double(),
           "Smallest gap considered for the lubrification force calculation. This value is multiplied by the smallest cell size");
 
+        prm.declare_entry(
+          "explicit contact impulsion",
+          "false",
+          Patterns::Bool(),
+          "Bool to enable or disable the explicit evaluation of the contact impulsion. This means that if that parameter is set to true, the DEM is only run once, and the contact impulsion obtained is used for all Newton's iterations.");
```

Comment:
As discussed, I think this could be reworded. Bringing the .rst explanation here would be a good approach. Same comment can be applied to the parameters that follow.